Minor edits for abstract and intro (#654)
greenelab/covid19-review: content/02.introduction.md

### Initial Characterization of SARS-CoV-2

The first genome sequence of the virus was released on January 3, 2020 and revealed that the cluster of pneumonia cases seen in Wuhan were caused by a novel coronavirus [@doi:10.46234/ccdcw2020.017].
Multiple research groups have drafted the genome sequence of SARS-CoV-2 based on sequences developed from clinical samples collected from the lower respiratory tract, namely bronchoalveolar lavage fluid (BALF), and the upper respiratory tract, in the form of throat swabs [@doi:10.1038/s41586-020-2008-3; @doi:10.1038/s41586-020-2012-7; @doi:10/ggjr43].
Analysis of the SARS-CoV-2 genome revealed significant sequence homology with two coronaviruses known to infect humans, with about 79% identity to SARS-CoV and 50% to MERS-CoV [@doi:10/ggjr43].
However, the highest degree of similarity was observed between SARS-CoV-2 and bat-derived SARS-like coronaviruses (bat-SL-CoVZC45 and bat-SL-CoVZXC21) [@doi:10/ggjr43; @doi:10.1038/s41586-020-2012-7], with identity between SARS-CoV-2 and RATG13 as high as 96.2% [@doi:10.1038/s41586-020-2012-7; @doi:10.1093/nsr/nwaa036].

> This evidence therefore suggests the SARS-CoV-2 virus may be the result of zoonotic transfer of a virus from bats to humans.

Author: ```suggestion
This evidence therefore suggests the SARS-CoV-2 virus is the result of zoonotic transfer of a virus from bats to humans.
```

These components of the immune system typically act together; the innate response acts first, and the adaptive response begins to act several days after initial infection following the clonal expansion of T and B cells [@doi:10.1016/j.jaci.2005.09.034].
After a virus enters into a host cell, its antigen is presented by major histocompatibility complex 1 (MHC 1) molecules and is then recognized by cytotoxic T lymphocytes.

> ##### Cytokine Storm

Author: This text moved to another section but was never deleted.

> One of the main immune responses contributing to the onset of Acute Respiratory Distress Syndrome (ARDS) in COVID-19 patients is the cytokine storm, which causes an extreme inflammatory response due to a release of pro-inflammatory cytokines and chemokines by immune effector cells.

Reviewer: ```suggestion
One of the main immune responses contributing to the onset of acute respiratory distress syndrome (ARDS) in COVID-19 patients is the cytokine storm, which causes an extreme inflammatory response due to a release of pro-inflammatory cytokines and chemokines by immune effector cells.
```
